server/src/test/java/org/opensearch/index/shard/RemoteStoreRefreshListenerTests.java

```diff
@@ -282,7 +282,6 @@ public void testRefreshSuccessOnSecondAttempt() throws Exception {
     /**
      * Tests retry flow after snapshot and metadata files have been uploaded to remote store in the failed attempt.
      * Snapshot and metadata files created in failed attempt should not break retry.
-     * @throws Exception
```

Comment:
needed to remove this otherwise compileTestJava failed with `no description for @throws` error.
```
> Task :server:compileTestJava
./OpenSearch/server/src/test/java/org/opensearch/index/shard/RemoteStoreRefreshListenerTests.java:285: warning: no description for @throws
     * @throws Exception
       ^
error: warnings found and -Werror specified
warnings found and -Werror specified
```